### PR TITLE
feat(WEB-8358): Literature pagination urls

### PR DESCRIPTION
**Pull request by rcmonsayac**
### What
- Change page param from query to path param
- Add link tags for `next `and `prev`

### CODE REVIEW

File: lib/literature/live/blog_live.ex [hunk 1]
```diff
@@ -91,17 +91,22 @@ defmodule Literature.BlogLive do
...
+      if with_page_number? do
+        params
+      else
+        Map.delete(params, "page")
+      end
+
+    if with_page_number? and params["page"] == "1" do
+      path
+      |> String.replace("/page/#{params["page"]}", "")
+      |> then(&{:noreply, push_navigate(socket, to: &1, replace: true)})
```
Comment (pau-riosa) [inline lib/literature/live/blog_live.ex:107]: nice usage of then here.

File: lib/literature/helpers/meta_tag_helpers.ex [hunk 2]
```diff
@@ -92,4 +92,45 @@ defmodule Literature.MetaTagHelpers do
...
     end
   end
+
+  @doc """
+  Render pagination link tags
+  """
+  def render_pagination_link_tags(
+        %{
+          live_action: :index,
+          page: %{page_number: page_number, total_pages: total_pages}
```
Comment (pau-riosa) [inline lib/literature/helpers/meta_tag_helpers.ex:102]: Are you using Scrivener here?

File: lib/literature/helpers/meta_tag_helpers.ex [hunk 3]
```diff
@@ -92,4 +92,45 @@ defmodule Literature.MetaTagHelpers do
...
+        },
+        current_url
+      ) do
+    cond do
+      page_number == 1 and total_pages > 1 ->
```
Comment (dgross881) [inline lib/literature/helpers/meta_tag_helpers.ex:107]: @rcmonsayac can we move these into smaller fucntions to make it more readable instead of cond

**Review by dgross881 — CHANGES_REQUESTED** [review]

File: lib/literature/helpers/meta_tag_helpers.ex (hunk 3 above)
Comment (rcmonsayac) [reply lib/literature/helpers/meta_tag_helpers.ex]: @dgross881 updated

**Review by dgross881 — APPROVED** [review]

**Pull request merged**